WidgetExtension/Up Next/UpNextStyles.swift

```diff
@@ -4,6 +4,128 @@ import SwiftUI
 let darkBackgroundColor = Color(UIColor.tertiarySystemFill)
 let lightBackgroundColor = Color(.clear)
 
+struct PCWidgetColorScheme {
+    let topBackgroundColor: Color
+    let bottomBackgroundColor: Color
+    let topButtonBackgroundColor: Color
+    let bottomButtonBackgroundColor: Color
+    let topTextColor: Color
+    let bottomTextColor: Color
+    let topButtonTextColor: Color // foreground color?
+    let bottomButtonTextColor: Color // foreground color?
+    let iconAssetName: String
+    let filterViewBackgroundColor: Color
+    let filterViewTextColor: Color
+    let filterViewIconAssetName: String
+}
+
+let rgbMax = 255.0
+let widgetRedDark = Color(.sRGB, red: 217/rgbMax, green: 32/rgbMax, blue: 28/rgbMax)
+let widgetRedLight = Color(.sRGB, red: 244/rgbMax, green: 62/rgbMax, blue: 55/rgbMax)
+let widgetBlack = Color(.sRGB, red: 22/rgbMax, green: 23/rgbMax, blue: 24/rgbMax)
+let widgetCoolGrey = Color(.sRGB, red: 41/rgbMax, green: 43/rgbMax, blue: 46/rgbMax)
+
+let widgetColorSchemeBold = PCWidgetColorScheme(
```

Comment:
Absolutely, I like that suggestion. Also makes using it way less verbose since I can just use `.bold` in places where a PCWidgetColorScheme is expected.